### PR TITLE
Version 0.4

### PR DESCRIPTION
**Pull request by marsofandrew**
rework sdk

Now we can create requests automatically, we only should write special template.
and we can execute our custom functions by using key: 'executed_by'

### CODE REVIEW

**Comment by dadlex** [comment]
```
+def do_query_request(unity, params, params_types, rest_object):
+    # if not validator.check_parameters(params,params_types):
+    #    supportive_functions.raise_exception_about_parameters(params_types)
```
Why the parameters check is omitted?

**Comment by dadlex** [comment]
Why validator.py is empty now? Hasn't it been merged with the Anton Yurkov's version?

File: dellemc_unity_sdk/runner.py [hunk 1]
```diff
@@ -1,65 +1,125 @@
...
+def do_update_request(unity, params, params_types, rest_object, action):
+    if not validator.check_parameters(params, params_types):
+        supportive_functions.raise_exception_about_parameters(params_types)
+    reply = unity.update(action, rest_object, params)
+    return reply
+
+
+def do_query_request(unity, params, params_types, rest_object):
+    # if not validator.check_parameters(params,params_types):
+    #    supportive_functions.raise_exception_about_parameters(params_types)
```
Comment (marsofandrew) [inline dellemc_unity_sdk/runner.py]: Now i don't know what parameters are required and optional

**Comment by marsofandrew** [comment]
validator.py isn't empty

**Pull request merged**